src/dev/flang/ast/AstErrors.java

```diff
@@ -132,9 +128,9 @@ static String sle(List<Expr> e)
   {
     return expr(e.toString());
   }
-  static String s(Stmnt s)
+  static String s(AbstractAssign a)
```

Comment:
There is an ambiguity problem because `Feature` inherits from `Stmnt` and `s(AbstractFeature af)` exists as well. I also think `s(Stmnt s)` which just uses s.toString() which isn't even an abstract method of `Stmnt` is rather useless and can lead to unintentional consequences like the `expr.toString()` problem.